### PR TITLE
Sanketdg/masterfix

### PR DESCRIPTION
**Pull request by SanketDG**

### CODE REVIEW

**Comment by SanketDG** [comment]
The `dockerfile_lint` fix was fixed in 0.2.4

File: tests/css/CSSAutoPrefixBearTest.py [hunk 1]
```diff
@@ -5,10 +5,10 @@
 good_file = """
 .example {
     display: -webkit-box;
-    display: -webkit-flex;
     display: -ms-flexbox;
     display: flex;
 }
+
```
Comment (sils1297) [inline tests/css/CSSAutoPrefixBearTest.py]: what's this line for?

**Comment by sils1297** [comment]
should we fix the version then?

File: tests/css/CSSAutoPrefixBearTest.py (hunk 1 above)
Comment (SanketDG) [inline tests/css/CSSAutoPrefixBearTest.py]: gah sorry, fixed.

**Comment by SanketDG** [comment]
> should we fix the version then?

Not needed now. `0.2.4` was released with the fix, so it can stay as it is.

**Comment by sils1297** [comment]
can you also cherrypick @adtac s commit with the new rultor secret file?

**Comment by sils1297** [comment]
ack https://github.com/coala/coala-bears/pull/832/commits/18defb41a5c280bf65b7bd30cce2c45b4e22de69 https://github.com/coala/coala-bears/pull/832/commits/249cd64eab17d888eb76a04cf096aadb83853c9d

**Comment by sils1297** [comment]
@SanketDG what's with that nondeterministically appearing coverage thingie?

**Comment by SanketDG** [comment]
> what's with that nondeterministically appearing coverage thingie?

Is it non-deterministic? I think it was failing because `DockerfileLintBear` and `CSSAutoPrefixBear` was failing, which is okay.

Doesn't seem to fail now.

**Comment by sils1297** [comment]
ah ok it felt nondeterministic I have no idea what I'm talking about - ignore me :P

**Comment by sils1297** [comment]
@SanketDG can you merge manually? git pull the master branch, do a fastforward merge and push it up. GitHub should accept it since all the CI has been run.

**Comment by SanketDG** [comment]
> can you merge manually?

Is this the correct set of steps?

```
git pull upstream master
git push upstream master
```

`upstream` is coala `upstream`

**Comment by SanketDG** [comment]
nvm,

I did `git merge --ff sanketdg/masterfix` on a updated master branch.

**Comment by sils1297** [comment]
well now do a git push ;)

**Comment by SanketDG** [comment]
```
$ git push upstream master
```

I get:

```
Total 0 (delta 0), reused 0 (delta 0)
remote: error: GH006: Protected branch update failed for refs/heads/master.
remote: error: You're not authorized to push to this branch. Visit https://help.github.com/articles/about-protected-branches/ for more information.
To git@github.com:coala/coala-bears.git
 ! [remote rejected] master -> master (protected branch hook declined)
error: failed to push some refs to 'git@github.com:coala/coala-bears.git'
```

Should I force push or something? Sorry I dont want to mess anything up :stuck_out_tongue:

**Comment by sils1297** [comment]
merged

**Pull request merged**